api/notes/noteRouter.js

```diff
@@ -54,10 +50,9 @@ router.post('/', authRequired, checkBodyIsComplete, async (req, res, next) => {
       content: req.body.content,
       level: req.body.level,
       visible_to_admin: req.body.visible_to_admin,
-      visible_to_moderator: req.body.visible_to_moderator,
       visible_to_mentor: req.body.visible_to_mentor,
-      profile_id_mentor: req.body.profile_id_mentor,
-      profile_id_mentee: req.body.profile_id_mentee,
+      mentor_id: req.body.mentor_id,
+      mentee_id: req.body.mentee_id,
```

Comment:
Why not keep the word profile in the actual table?  This does look more efficient and organized though, so it looks good!

Reply:
The initial reason behind removing "profile" and going back to "mentor_id" and "mentee_id" is to remain consistent with migrations housing these similar columns across the backend and to reduce overall keystrokes as the website expands.